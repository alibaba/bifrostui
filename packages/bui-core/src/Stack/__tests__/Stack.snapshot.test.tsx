import { snapshotTest } from 'testing';

describe('Stack snapshot', () => {
  snapshotTest('Stack');
});
