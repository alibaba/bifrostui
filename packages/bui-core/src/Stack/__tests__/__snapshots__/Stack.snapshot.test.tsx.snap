// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stack snapshot Stack demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "center",
        "--flex-direction": "column",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "0",
        "height": "150px",
        "width": "150px",
      }
    }
    wrap=""
  >
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      type="button"
    >
      按钮1
    </button>
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      type="button"
    >
      按钮2
    </button>
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      type="button"
    >
      按钮3
    </button>
  </div>
  <div
    className="bui-stack-demo-options"
  >
    <div
      className="bui-radio-group"
    >
      <div
        className="bui-stack-demo-options-title"
      >
        direction
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="row"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           row
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="row-reverse"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           row-reverse
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="column"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          column
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="column-reverse"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          column-reverse
        </div>
      </div>
    </div>
    <div
      className="bui-radio-group"
    >
      <div
        className="bui-stack-demo-options-title"
      >
         justifyContent
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="flex-start"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           flex-start
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="flex-end"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           flex-end
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="center"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          center
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="space-between"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          space-between
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="space-around"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          space-around
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="space-evenly"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          space-evenly
        </div>
      </div>
    </div>
    <div
      className="bui-radio-group"
    >
      <div
        className="bui-stack-demo-options-title"
      >
         alignItems
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="flex-start"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           flex-start
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="flex-end"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           flex-end
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="center"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          center
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="stretch"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          stretch
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="baseline"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           baseline
        </div>
      </div>
    </div>
    <div
      className="bui-radio-group"
    >
      <div
        className="bui-stack-demo-options-title"
      >
         flexWrap
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="wrap"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           wrap
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="no-wrap"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           no-wrap
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="wrap-reverse"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          wrap-reverse
        </div>
      </div>
    </div>
    <div
      className="bui-radio-group"
    >
      <div
        className="bui-stack-demo-options-title"
      >
         spacing
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="10px"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          10px
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="20px"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           20px
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="30px"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          30px
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="40px"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
          40px
        </div>
      </div>
      <div
        className="bui-radio bui-radio-label-right"
        onClick={[Function]}
      >
        <input
          checked={false}
          className="bui-radio-input"
          disabled={false}
          onChange={[Function]}
          type="radio"
          value="50px"
        />
        <div
          className="bui-radio-icon"
        >
          <span
            className="bui-radio-btn-icon"
          >
            <svg
              className="bui-svg-icon bui-radio-btn-icon-outline icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M48 88C25.9 88 8 70.1 8 48S25.9 8 48 8s40 17.9 40 40-17.9 40-40 40zm0-6c18.8 0 34-15.2 34-34S66.8 14 48 14 14 29.2 14 48s15.2 34 34 34z\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
            <svg
              className="bui-svg-icon bui-radio-btn-icon-contained icon-size-medium"
              color="#959aa5"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<path d=\\"M28 48a20 20 0 1 0 40 0 20 20 0 1 0-40 0\\"/>",
                }
              }
              focusable="false"
              viewBox="0 0 96 96"
            />
          </span>
        </div>
        <div
          className="bui-radio-label"
        >
           50px
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Stack snapshot Stack demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    className="bui-stack"
    style={
      Object {
        "--align-items": "center",
        "--flex-direction": "column",
        "--flex-wrap": "nowrap",
        "--justify-content": "center",
        "--spacing": "10px",
        "width": "100px",
      }
    }
  >
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      type="button"
    >
      按钮1
    </button>
    <div
      className="bui-divider bui-divider-horizontal"
      style={Object {}}
    />
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      type="button"
    >
      按钮2
    </button>
    <div
      className="bui-divider bui-divider-horizontal"
      style={Object {}}
    />
    <button
      className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
      type="button"
    >
      按钮3
    </button>
  </div>
</div>
`;
